components/board/pi/impl/board.go

```diff
@@ -55,11 +55,7 @@ func init() {
 				conf resource.Config,
 				logger golog.Logger,
 			) (board.Board, error) {
-				boardConfig, err := resource.NativeConfig[*genericlinux.Config](conf)
-				if err != nil {
-					return nil, err
-				}
-				return NewPigpio(ctx, conf.ResourceName(), boardConfig, logger)
+				return NewPigpio(ctx, conf.ResourceName(), conf, logger)
```

Comment:
Is there a reason for the creator to remain exported since you're refactoring already?

Reply:
:eyes: No! I had assumed it was exported so it could be used in unit tests, but they're all in the same package name already. Good call!